app/theme/client/imports/components/messages.css

```diff
@@ -205,7 +205,7 @@
 	.message {
 		&:hover,
 		&.active {
-			background-color: rgba(15, 34, 0, 0.05);
+			background-color: #F7F8FA;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected "#F7F8FA" to be "#f7f8fa" (color-hex-case)](https://app.codacy.com/manual/RocketChat/Rocket-Chat/pullRequest?prid=5416119)